packages/esbuild/test/index.js

```diff
@@ -365,7 +366,7 @@ test('@mdx-js/esbuild', async function (t) {
               file: 'test/esbuild.mdx',
               length: 11,
               line: 3,
-              lineText: '# Hello, <Message />',
+              lineText: '',  // Not available when place.offset is deleted
```

Comment:
```suggestion
              lineText: '',
```

No comment needed, the code below for `6` is clear that the test is about missing offsets